applications/cm-service/templates/pvc.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: cm-service-output
+  labels:
+    {{- include "cm-service.labels" . | nindent 4 }}
+spec:
+  {{- with .Values.config.outputVolume.storageClassName }}
```

Comment:
What's the advantage of doing a `with` instead of just `storageClassName: {{ .Values.config.outputVolume.storageClassName }}` ?

Reply:
The body of the `with` is not expanded if the variable is `null`, so you don't get a dangling/empty yaml key this way.